internal/client/datadog/client_test.go

```diff
@@ -362,7 +362,7 @@ var _ = Describe("DatadogClient", func() {
 			return len(bodies)
 		}
 
-		Eventually(f).Should(BeNumerically(">", 1))
+		Eventually(f).Should(BeNumerically("==", 0))
```

Comment:
Why this ? If we're splitting packets, then we should eventually have several `bodies`